cli/hzn.go

```diff
@@ -566,6 +567,9 @@ Environment Variables:
 	mmsObjectPublishPat := mmsObjectPublishCmd.Flag("pattern", msgPrinter.Sprintf("If you want the object to be deployed on nodes using a given pattern, specify it using this flag. This flag is optional and can only be used with --type and --id. It is mutually exclusive with -m")).Short('p').String()
 	mmsObjectPublishDef := mmsObjectPublishCmd.Flag("def", msgPrinter.Sprintf("The definition of the object to publish. A blank template can be obtained from the 'hzn mss object new' command. Specify -m- to read from stdin.")).Short('m').String()
 	mmsObjectPublishObj := mmsObjectPublishCmd.Flag("object", msgPrinter.Sprintf("The object (in the form of a file) to publish. This flag is optional so that you can update only the object's definition.")).Short('f').String()
+	mmsObjectPublishSkipDigitalSig := mmsObjectPublishCmd.Flag("skipDigitalSig", msgPrinter.Sprintf("Indicate no data signature applied to the object data (the file passed in with -f flag). It is mutually exclusive with -s")).Bool()
+	mmsObjectPublishDSHashAlgo := mmsObjectPublishCmd.Flag("hashAlgo", msgPrinter.Sprintf("The hash algorithm used for hash generation and digital signing. Supported hash algorithms are SHA1, SHA256. Default is SHA1. It is mutually exclusive with --skipDigitalSig")).Short('a').String()
```

Comment:
Change description to: "The hash algorithm used to hash the object data before signing it, ensuring data integrity during upload and download. Supported hash algorithms are SHA1 or  SHA256, the default is SHA1. It is mutually exclusive with the --noIntegrity flag."